shared/templates/template_OVAL_service_disabled

```diff
@@ -278,4 +280,6 @@
 
 {{% endif %}}
 
+{{{ oval_test_package_removed(package=PACKAGENAME, test_id=package_removed_test_id) }}}
+
```

Comment:
The template template_OVAL_service_enabled has a similar problem which needs to be addressed.

Reply:
Should be fixed in 2c0f8e3